primap2/pm2io/_data_reading.py

```diff
@@ -1490,64 +1523,105 @@ def harmonize_units(
     if unit_col is None:
         unit_col = dim_aliases.get("unit", "unit")
 
+    # find basic entities for all entities and make a list
     entities = data[entity_col].unique()
+    basic_entities = {}
+
     # print(entities)
     for entity in entities:
         # check if GWP given in entity
+        # print(f"entity: {entity}")
```

Comment:
```suggestion
        logger.debug(f"entity: {entity}")
```

https://primap2.readthedocs.io/en/stable/development.html#logging